repos/system_upgrade/el7toel8/actors/initramfs/tmpactor/actor.py

```diff
@@ -0,0 +1,22 @@
+from leapp.actors import Actor
+from leapp.models import TargetInitramfsTasks
+from leapp.tags import TargetTransactionChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckInitramfsTasks(Actor):
```

Comment:
This "tmp" actor is supposed to be used for testing or is it a leftover?

Reply:
I expected originally to remove this actor once we fix the sanity checks: https://github.com/oamg/leapp/pull/680 The most probably we will remove the actor before the release. But to keep the stuff simle, I think we should merge this PR with the tmp actor as well. I am going to update the name and description to make clear the right purpose of the actor.

Reply:
updated